src/main/java/org/sagebionetworks/web/client/jsinterop/SRC.java

```diff
@@ -15,14 +15,14 @@
 		public static ReactFunctionComponent<EvaluationEditorPageProps> EvaluationEditorPage;
 		public static ReactFunctionComponent<AccessTokenPageProps> AccessTokenPage;
 		public static ReactFunctionComponent<EmptyProps> DownloadCartPage;
-		public static ReactFunctionComponent<ShowDownloadV2Props> ShowDownloadV2;
```

Comment:
New nav has it's own Download Cart link, so this component is no longer used in the project.